packages/x-components/tests/e2e/cucumber/identifier-results/identifier-results.feature

```diff
@@ -22,7 +16,7 @@ Feature: Identifier results component
   Scenario Outline: 2. ID search with no results is made
```

Comment:
The `2` and `3` scenarios will still pass if a request is made and it returns results, as the `Then no identifier results are displayed` is executed immediately after typing, and at that moment there will never be any painted results. I suggest combining it with the happy path with results, and assert that the identifier results are reset in both cases.

```feature
  Scenario: 1. ID search shows results and clears them when a non id query is made
    Given an ID results API with a known response
    And   start button is clicked
    When  a "01" with results is typed
    Then  identifier results are displayed
    When  a "lego" with results is typed
    Then  no identifier results are displayed


  Scenario: 2. ID search shows results and clears them when a id query with no results is made
    Given an ID results API with a known response
    And   start button is clicked
    When  a "01" with results is typed
    Then  identifier results are displayed
    Given an ID results API with no results
    When  a "23" with results is typed
    Then  no identifier results are displayed
```

Reply:
changed